IngresseSDK/Services/EventService.swift

```diff
@@ -55,4 +62,91 @@ public class EventService: BaseService {
             onError(error)
         }
     }
+
+    /// Get event list from state and category with search term
+    ///
+    /// - Parameters:
+    ///   - page: Index of page to get
+    ///   - place: Abbreviation of the state
+    ///   - category: ID of the category
+    ///   - searchTerm: Term to filter event list
+    ///   - delegate: Callback listener
+    public func getEvents(page: Int,
+        from place: String,
+        ofCategory category: Int,
+        andSearchTerm searchTerm: String,
+        delegate: EventSyncDelegate) {
+        let path = categories[category]
+
+        let fields = "id,title,planner,date,venue,rsvp,saleEnabled,type,link,poster,description"
```

Comment:
Not mandatory, but I would add the fields as an additional, not required, parameter with the fields below as default.

Please, see feature/owls branch as reference. I believe we have a method like this out there.

Reply:
the parameter "fields" is not required to call this function, as you can see on its header, these fields are requested by default